models/RwdImage.php

```diff
@@ -89,6 +115,9 @@ public function picture( $size, $attributes = array() ) {
 		if ( ! $this->attachment ) {
 			return '';
 		}
+		if ( $this->is_svg_image() ) {
+			return $this->get_svg_image( $this->attachment->ID );
+		}
```

Comment:
you should just allow to print this image in configured image sizes, you should not terminate method execution